mantidimaging/gui/windows/spectrum_viewer/view.py

```diff
@@ -92,7 +97,47 @@ def __init__(self, main_window: 'MainWindowView'):
         self.tableView.setSelectionMode(QAbstractItemView.SingleSelection)
         self.tableView.setAlternatingRowColors(True)
 
+        # Roi Prop table
+        self.roiPropertiesGroupBoxTitle: str = "Roi Properties: "
+        self.roi_table_properties = ["Top", "Bottom", "Left", "Right"]
+        self.roi_table_properties_secondary = ["Width", "Height", "Area"]
```

Comment:
If @samtygier-stfc and maybe IMAT agree, i'd consider removing area if not needed to free up some space in the GUI.